### ISSUE TITLE
V2: Actions appearing blank for duplicate logics on logic tab

### ISSUE THREAD

**Issue by AbhishekKS** [surveyjs/survey-creator#2083]
### Are you requesting a feature, reporting a bug or ask a question?

Bug

### What is the current behavior?
When I provide duplicate logic on the logic tab, after that when I open the clubbed logic, actions appear blank. Seems to work fine in V1.

### What is the expected behavior?
Action should be populated

### How would you reproduce the current behavior (if this is a bug)?
Sample JSON to replicate this issue:
{
 "title": "Survey Title",
 "pages": [
  {
   "name": "page1",
   "elements": [
    {
     "type": "text",
     "name": "question1"
    }
   ]
  }
 ],
 "triggers": [
  {
   "type": "complete",
   "expression": "{question1} = 'yes'"
  },
  {
   "type": "complete",
   "expression": "{question1} = 'yes'"
  },
  {
   "type": "complete",
   "expression": "{question1} = 'no'"
  },
  {
   "type": "complete",
   "expression": "{question1} = 'no'"
  }
 ]
}

**In V2:**
![image](https://user-images.githubusercontent.com/13827554/138043533-a7139318-0905-4678-a9d8-6012cac4dbdc.png)

**In V1:**
![image](https://user-images.githubusercontent.com/13827554/138043633-3dd86f3c-c2f0-4167-bf26-c7525b7c01d7.png)

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://surveyjstest.azurewebsites.net//Examples/CreatorSinglePage?id=options&platform=ReactjsV2&theme=modern

Test code

```
your_code_here

```

### Specify your

* browser: Chrome 94
* editor version: Seems existing issue in V2

**Comment by andrewtelnov** [comment]
@AbhishekKS I was able to reproduce the issue. We will work on the fix.

Thank you,
Andrew

**Comment by andrewtelnov** [comment]
@AbhishekKS After some investigation I can say that it is not a bug. You can't have more then one complete trigger for one expression, it doesn't make sence. That is why on "complete" trigger is gone.
I can write the code that will remove one "complete" trigger add do not add it into logic initially. I am not sure, do we need this code or not?

Thank you,
Andrew

**Comment by AbhishekKS** [comment]
Hi @andrewtelnov,
I agree, it's not a valid use case where the user repeats the same trigger and expression. It does not make sense. I just came across it randomly while self-testing. It's just when I compared it with V1, I found a different visual behavior. So thought of bringing this to your kind notice. Anyways, the user can always remove duplicates if he/she ever comes across such a case. In my opinion, we can live with the current behavior.

Thanks
Abhishek
